internal/config/canary_config.go

```diff
@@ -48,7 +49,8 @@ const (
 	DynamicConfigFileEnvVar             = "DYNAMIC_CONFIG_FILE"
 	DynamicConfigWatcherIntervalEnvVar  = "DYNAMIC_CONFIG_WATCHER_INTERVAL"
 	//TODO: This will be removed when Support the OTEL_TRACES_EXPORTER env var is available in the SDK see: https://github.com/open-telemetry/opentelemetry-go/issues/2310
-	ExporterTypeTracing = "EXPORTER_TYPE_TRACING"
+	ExporterTypeTracing           = "EXPORTER_TYPE_TRACING"
+	PrometheusConsantLabelsEnvVar = "PROMETHEUS_CONSTANT_LABELS"
```

Comment:
please align with previous assignments

Reply:
So this is actually what go fmt enforces. The comment above breaks the alignment.

Reply:
Isn't it possible to force the alignment we want?

Reply:
I don't think so no, go fmt is pretty strict.

Reply:
at this point, why it was ok before? I don't remember I am using go fmt.

Reply:
Maybe this is due to using a newer version of go fmt (1.19 I think)? If you remove the comment the envars line up with the ones above.

Reply:
can we move that comment on the right of line 52 after the declaration? It should align then, right?

Reply:
Yes that should work